### DISCUSSION TITLE
Error from SEQ when authenticating with Azure Active directory

### DISCUSSION THREAD

**Discussion by juan-ecommerce** [datalust/seq-tickets#1638] [Help] [answered]
Hi
I have configure to login to SEQ with Azure Active Directory.
For me is working perfectly but when my anyone on my team is login, they are getting "Could not sign in to Seq" "No access to this instance has been granted."
Any idea what I did wrong?
![image](https://user-images.githubusercontent.com/36268543/185163437-f32200b5-48af-44a2-910e-2916484aeecd.png)

The error message shows when a user has already login to Azure AD.

Tanks

**Comment by nblumhardt** [comment, marked as answer]
Hi Juan,

Thanks for getting in touch. There are two ways to resolve this.

1. If you want all users granted access to the app in your AAD instance to be able to log in, select "Automatically provision user-level accounts" on the Settings > System page in Seq
2. Otherwise, you need to add users one by one in Settings > Users; Seq will match them to their AAD identities by username (i.e. an email address) the first time they log in.

Let me know if this helps, or if you need any more info.

Best regards,
Nick

**Reply by juan-ecommerce** [reply]
Thanks
